scripts/download_backtest_data.py

```diff
@@ -15,6 +15,9 @@
 
 TICKER_INTERVALS = ['1m', '5m']
 PAIRS = []
+MIN_SECCONDS = 60
+HOUR_SECCONDS = 60 * MIN_SECCONDS
+DAY_SECCONDS = 24 * HOUR_SECCONDS
```

Comment:
seconds may seem long sometimes - but they did not earn a 2nd C yet ;)

Reply:
=))) yeah! Just wrote SEC and then added CONDS=)

Will do all the fixes later today